openlcs/tasks/models.py

```diff
@@ -43,6 +43,19 @@ class Task(models.Model):
         blank=True, null=True,
         on_delete=models.CASCADE
     )
+    # TODO(Weixin Bu): modify it later
```

Comment:
If there are changes on the model, migrations should be attached, please mark this as draft if this PR is still in progress.

Reply:
OK. The migrations file is submitted here: `openlcs/tasks/migrations/0003_auto_20221118_0247.py`.
The PR is in progress.